generator/internal/parser/protobuf.go

```diff
@@ -437,11 +442,13 @@ func processMethod(state *api.APIState, m *descriptorpb.MethodDescriptorProto, m
 		return nil
 	}
 	method := &api.Method{
-		ID:           mFQN,
-		PathInfo:     pathInfo,
-		Name:         m.GetName(),
-		InputTypeID:  m.GetInputType(),
-		OutputTypeID: m.GetOutputType(),
+		ID:                  mFQN,
+		PathInfo:            pathInfo,
+		Name:                m.GetName(),
+		InputTypeID:         m.GetInputType(),
+		OutputTypeID:        m.GetOutputType(),
+		ClientSideStreaming: m.ClientStreaming != nil && *m.ClientStreaming,
```

Comment:
```suggestion
		ClientSideStreaming: m.GetClientStreaming(),
```

Reply:
Done.  Maybe I will learn the Proto -> Golang mapping one of these days.